homeassistant/components/drop_connect/coordinator.py

```diff
@@ -69,9 +66,9 @@ async def DROPMessageReceived(
 
     # Device properties
     @property
-    def id(self) -> str:
-        """Return DROP device id."""
-        return self._drop_device_id
+    def unique_id(self) -> str:
+        """Return device unique id."""
+        return self.config_entry.unique_id or ""
```

Comment:
It can't be empty, but mypy complains if the return value isn't a string. Perhaps it would be better to remove the `or ""` and instead define the function as `def unique_id(self) -> str | None:`?

Reply:
it seems the `device_id` property is only used for the deviceinfo of the sensor entities?
than we can remove this property at all and use `coordinator.config_entry.unique_id` instead